code/modules/mob/living/emote.dm

```diff
@@ -253,6 +253,12 @@
 	message = "shakes their head."
 	emote_type = EMOTE_AUDIBLE
 
+/datum/emote/living/shiver
+	key = "shiver"
+	key_third_person = "shiver"
+	message = "shivers."
+	emote_type = EMOTE_AUDIBLE

```

Comment:
probably the "brrrr" noise people sometimes make when they shiver, at a guess.
that or it's purely for mechanical reasons (sight/deaf checks).